src/generator.js

```diff
@@ -4,7 +4,7 @@ import {
   concat,
   length,
   map,
-  merge,
+  mergeRight,
```

Comment:
Was `merge` actually used? I'm just surprised to see the import change with no other changes in the file.

Reply:
It wasn't for this file. It is used in other files though! I removed it here.